uint8 id
uint8 time_interval 